wdio.conf.js

```diff
@@ -90,6 +90,12 @@ exports.config = {
     recents: [
       './test/journeys/specs/recents/**/*.js'
     ],
+    recentsGlobal: [
+      './test/journeys/specs/recents/global/**/*.js'
+    ],
+    recentsData: [
+      './test/journeys/specs/recents/dataApi/**/*.js'
+    ],
```

Comment:
What's the reasoning for these extra suites? Just wondering as we don't usually do that

Reply:
laziness for local testing